### PR TITLE
Add metrics collection and metrics-based dashboards info to HAProxy Otel app

### PR DESCRIPTION
**Pull request by yasar-sumologic**
…Tel app doc

## Purpose of this pull request

Changes to haproxy-otel app doc after changes in it to add metric collection and metric-based dashboards

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 1]
```diff
@@ -29,12 +27,21 @@ The HAProxy logs are generated in files as configured in the configuration file
 
 Following are the [Fields](/docs/manage/fields/) which will be created as part of HAProxy App install if not already present.
 
-- `webengine.cluster.name`. User configured. Enter a name to identify the Haproxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
-- `webengine.system`. Has fixed value of **haproxy**
-- `sumo.datasource`. Has fixed value of **haproxy**
+- **`webengine.cluster.name`**. User configured. Enter a name to identify the Haproxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
```
Comment (sumoanema) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:30]: Can you add deployment.environment here

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 2]
```diff
@@ -97,6 +104,12 @@ import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
 
 In this step, you will configure the yaml required for HAProxy Collection.
 
+Below are the inputs required:
+
+- **Endpoint**. The URL of the httpd status endpoint (default: `http://localhost:8404/stats`).
+- **HAProxy logs Path**. Enter the path to the log file for your HAProxy instance.
+- **Fields**. `webengine.cluster.name`.
+
 The path of the log file configured to capture haproxy logs is needed to be given here.
```
Comment (sumoanema) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:113]: This is already covered above under HAProxy log Path

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 3]
```diff
@@ -97,6 +104,12 @@ import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
 
 In this step, you will configure the yaml required for HAProxy Collection.
 
+Below are the inputs required:
+
+- **Endpoint**. The URL of the httpd status endpoint (default: `http://localhost:8404/stats`).
+- **HAProxy logs Path**. Enter the path to the log file for your HAProxy instance.
+- **Fields**. `webengine.cluster.name`.
```
Comment (sumoanema) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:111]: This line is not making any sense. Can we remove it or provide more information

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 4]
```diff
@@ -197,6 +212,34 @@ webengine.cluster.name=* %"sumo.datasource"=haproxy
 | limit 5
 ```
 
+### Metrics
+
+Here's a sample Metrics query from the **Http Response Codes** dashboard > **HAProxy - Backend Metrics** panel:
+
+```
+sumo.datasource=haproxy metric=haproxy.requests.total status_code=* haproxy.service_name=backend deployment.environment={{deployment.environment}} webengine.cluster.name={{webengine.cluster.name}} webengine.node.name={{webengine.node.name}}  haproxy.proxy_name={{haproxy.proxy_name}}
```
Comment (sumoanema) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:220]: this should be a resolved query. That is all the parameters which are inside {{}} should be replaced with *

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 5]
```diff
@@ -11,11 +11,9 @@ import TabItem from '@theme/TabItem';
 
 <img src={useBaseUrl('img/integrations/web-servers/haproxy.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
 
-[HAProxy](https://docs.haproxy.org/2.6/intro.html) is open source software that provides a high availability load balancer and proxy server for TCP and HTTP-based applications that spreads requests across multiple servers.
+[HAProxy](https://docs.haproxy.org/2.6/intro.html) app is a unified logs and metrics app that helps you monitor the availability, performance, health, and resource utilization of HAProxy server farms. Preconfigured dashboards and searches provide visibility into your environment for real-time or historical analysis: visitor locations, HTTP error codes percentage, backend and frontend server statistics, traffic patterns, errors, server operations, and access from known malicious sources.
```
Comment (kimsauce) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:14]:
```suggestion
[HAProxy](https://docs.haproxy.org/2.6/intro.html) app is a unified logs and metrics app designed to help monitor the availability, performance, health, and resource utilization of HAProxy server farms. It provides preconfigured dashboards and searches that offer visibility into various aspects of your environment for both real-time and historical analysis: visitor locations, HTTP error codes percentage, backend and frontend server statistics, traffic patterns, errors, server operations, and access from known malicious sources.
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 6]
```diff
@@ -11,11 +11,9 @@ import TabItem from '@theme/TabItem';
...
 
-The Sumo Logic app for HAProxy helps you monitor activity in HAProxy. The preconfigured dashboards provide information about site visitors, including location of visitors, HTTP Error codes percentage, Backend and Frontend server statistics.
-
-HAProxy logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+The OpenTelemetry collector runs on the same host as HAProxy and uses the [HAProxy Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/haproxyreceiver) and the [Sumo Logic OpenTelemetry Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sumologicexporter) to send the metrics to Sumo Logic.HAProxy logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
```
Comment (kimsauce) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:16]:
```suggestion
The OpenTelemetry collector runs on the same host as HAProxy and uses the [HAProxy Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/haproxyreceiver) and the [Sumo Logic OpenTelemetry Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sumologicexporter) to send the metrics to Sumo Logic. HAProxy logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 1 above)
Comment (kimsauce) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:30]:
```suggestion
- **`webengine.cluster.name`**. User configured. Enter a name to identify the HAProxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 7]
```diff
@@ -29,12 +27,21 @@ The HAProxy logs are generated in files as configured in the configuration file
...
+- **`webengine.cluster.name`**. User configured. Enter a name to identify the Haproxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`webengine.system`**. Has fixed value of **haproxy**
+- **`sumo.datasource`**. Has fixed value of **haproxy**
+- **`webengine.node.name`**. Has the value of host name of the machine which is being monitored
 
 ## Prerequisites
 
+### For metrics collection
+
+The receiver used gets stats from an HAProxy instance using the `stats` endpoint. This receiver supports HAProxy version 2.3.9+.
```
Comment (kimsauce) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:39]:
```suggestion
The receiver used gets stats from an HAProxy instance using the `stats` endpoint. This receiver supports HAProxy version 2.3.9 and later.
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 8]
```diff
@@ -197,6 +212,34 @@ webengine.cluster.name=* %"sumo.datasource"=haproxy
...
+
+Here's a sample Metrics query from the **Http Response Codes** dashboard > **HAProxy - Backend Metrics** panel:
+
+```
+sumo.datasource=haproxy metric=haproxy.requests.total status_code=* haproxy.service_name=backend deployment.environment={{deployment.environment}} webengine.cluster.name={{webengine.cluster.name}} webengine.node.name={{webengine.node.name}}  haproxy.proxy_name={{haproxy.proxy_name}} 
+| parse field=status_code *  as code 
+| avg by webengine.cluster.name,webengine.node.name,haproxy.proxy_name,code
+```
+
+## Sample Metrics
```
Comment (kimsauce) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:225]:
```suggestion
## Sample metrics
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 2 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 3 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: removed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 4 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 9]
```diff
@@ -11,11 +11,9 @@ import TabItem from '@theme/TabItem';
 
 <img src={useBaseUrl('img/integrations/web-servers/haproxy.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
 
-[HAProxy](https://docs.haproxy.org/2.6/intro.html) is open source software that provides a high availability load balancer and proxy server for TCP and HTTP-based applications that spreads requests across multiple servers.
+The [HAProxy](https://docs.haproxy.org/2.6/intro.html) app is a unified logs and metrics app designed to help monitor the availability, performance, health, and resource utilization of HAProxy server farms. It provides preconfigured dashboards and searches that offer visibility into your environment for both real-time and historical analysis: visitor locations, HTTP error codes percentage, backend and frontend server statistics, traffic patterns, errors, server operations, and access from known malicious sources.
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:14]:
```suggestion
The [HAProxy](https://docs.haproxy.org/2.6/intro.html) app is a unified logs and metrics app designed to help you monitor the availability, performance, health, and resource utilization of HAProxy server farms. It provides preconfigured dashboards and searches that offer visibility into your environment for real-time and historical analysis: visitor locations, HTTP error codes percentage, backend and frontend server statistics, traffic patterns, errors, server operations, and access from known malicious sources.
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 10]
```diff
@@ -11,11 +11,9 @@ import TabItem from '@theme/TabItem';
...
 
-The Sumo Logic app for HAProxy helps you monitor activity in HAProxy. The preconfigured dashboards provide information about site visitors, including location of visitors, HTTP Error codes percentage, Backend and Frontend server statistics.
-
-HAProxy logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+The OpenTelemetry collector runs on the same host as HAProxy and uses the [HAProxy Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/haproxyreceiver) and the [Sumo Logic OpenTelemetry Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sumologicexporter) to send the metrics to Sumo Logic. HAProxy logs are sent to Sumo Logic through the OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:16]:
```suggestion
The OpenTelemetry collector runs on the same host as HAProxy, where it uses the [HAProxy Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/haproxyreceiver) and the [Sumo Logic OpenTelemetry Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sumologicexporter) to send the metrics to Sumo Logic. HAProxy logs are sent to Sumo Logic through the OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 11]
```diff
@@ -29,12 +27,22 @@ The HAProxy logs are generated in files as configured in the configuration file
 
 Following are the [Fields](/docs/manage/fields/) which will be created as part of HAProxy App install if not already present.
 
-- `webengine.cluster.name`. User configured. Enter a name to identify the Haproxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
-- `webengine.system`. Has fixed value of **haproxy**
-- `sumo.datasource`. Has fixed value of **haproxy**
+- **`sumo.datasource`**. Has fixed value of **haproxy**
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:30]:
```suggestion
- **`sumo.datasource`**. Has fixed value of **haproxy**.
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 12]
```diff
@@ -29,12 +27,22 @@ The HAProxy logs are generated in files as configured in the configuration file
...
+- **`webengine.system`**. Has fixed value of **haproxy**
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:31]:
```suggestion
- **`webengine.system`**. Has fixed value of **haproxy**.
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 13]
```diff
@@ -210,6 +251,24 @@ Use this dashboard to:
...
 
+### Backend
+
+The **HAProxy - Backend** dashboard provides an at-a-glance view for the number of backend active servers, backend weight, respond code from backend and throughput http.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/HAProxy-OpenTelemetry/HAProxy-Backend.png' alt="Error Log Analysis" />
+
+### Frontend
+
+The **HAProxy - Backend** dashboard provides an at-a-glance view detail of HAProxy Frontend. It provides information such as number request to frontend, number of error requests, and current session.
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:262]:
```suggestion
The **HAProxy - Frontend** dashboard provides details of HAProxy Frontend. It provides information such as number request to frontend, number of error requests, and current session.
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 14]
```diff
@@ -210,6 +251,24 @@ Use this dashboard to:
...
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/HAProxy-OpenTelemetry/HAProxy-Frontend.png' alt="Error Log Analysis" />
+
+### Server
+
+The **HAProxy - Backend** dashboard provides an at-a-glance view detail of HAProxy Server. This dashboard helps you monitoring uptime, and error request by proxy.
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:268]:
```suggestion
The **HAProxy - Server** dashboard provides details of HAProxy Server. This dashboard helps you monitor the uptime and error request by proxy.
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 15]
```diff
@@ -197,6 +210,34 @@ webengine.cluster.name=* %"sumo.datasource"=haproxy
 | limit 5
 ```
 
+### Metrics
+
+Here's a sample Metrics query from the **Http Response Codes** dashboard > **HAProxy - Backend Metrics** panel:
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:215]:
```suggestion
Here is a sample metrics query from the **Http Response Codes** dashboard > **HAProxy - Backend Metrics** panel:
```

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 16]
```diff
@@ -97,9 +105,12 @@ import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
...
 In this step, you will configure the yaml required for HAProxy Collection.
 
-The path of the log file configured to capture haproxy logs is needed to be given here.
+Below are the inputs required:
+
+- **Endpoint**. The URL of the httpd status endpoint (default: `http://localhost:8404/stats`).
+- **HAProxy logs Path**. Enter the path to the log file for your HAProxy instance.
 
-The files are typically located in `/var/log/haproxy*.log`. If you're using a customized path, check the haproxy.conf file for this information. You can add any custom fields which you want to tag along with the data ingested in Sumo. Click on the **Download YAML File** button to get the yaml file.
+The path of the log file configured to capture haproxy logs is needed to be given here. The files are typically located in `/var/log/haproxy*.log`. If you're using a customized path, check the haproxy.conf file for this information. You can add any custom fields which you want to tag along with the data ingested in Sumo. Click on the **Download YAML File** button to get the yaml file.
```
Comment (JV0812) [inline docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md:113]:
```suggestion
The path of the log file configured to capture haproxy logs is needed to be given here. The files are typically located in `/var/log/haproxy*.log`. If you're using a customized path, check the `haproxy.conf` file for this information. You can add any custom fields which you want to tag along with the data ingested in Sumo Logic. Click on the **Download YAML File** button to get the yaml file.
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fixes

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 16 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 15 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: file updated.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 17]
```diff
@@ -210,6 +251,24 @@ Use this dashboard to:
 
 <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/HAProxy-OpenTelemetry/HAProxy-Overview.png' alt="Overview" />
 
+### Backend
+
+The **HAProxy - Backend** dashboard provides an at-a-glance view for the number of backend active servers, backend weight, respond code from backend and throughput http.
```
[the remark this answers is not in the document]
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: changes updated.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 14 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: changes updated.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 13 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: updated changes.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 18]
```diff
@@ -29,12 +27,22 @@ The HAProxy logs are generated in files as configured in the configuration file
...
 Following are the [Fields](/docs/manage/fields/) which will be created as part of HAProxy App install if not already present.
 
-- `webengine.cluster.name`. User configured. Enter a name to identify the Haproxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
-- `webengine.system`. Has fixed value of **haproxy**
-- `sumo.datasource`. Has fixed value of **haproxy**
+- **`sumo.datasource`**. Has fixed value of **haproxy**
+- **`webengine.system`**. Has fixed value of **haproxy**
+- **`webengine.cluster.name`**. User configured. Enter a name to identify the HAProxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`webengine.node.name`**. Has the value of host name of the machine which is being monitored
+- **`deployment.environment`**. User configured. This is the deployment environment where the Memcache cluster resides. For example: dev, prod or qa.
```
[the remark this answers is not in the document]
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md [hunk 19]
```diff
@@ -29,12 +27,22 @@ The HAProxy logs are generated in files as configured in the configuration file
 
 Following are the [Fields](/docs/manage/fields/) which will be created as part of HAProxy App install if not already present.
 
-- `webengine.cluster.name`. User configured. Enter a name to identify the Haproxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
-- `webengine.system`. Has fixed value of **haproxy**
-- `sumo.datasource`. Has fixed value of **haproxy**
+- **`sumo.datasource`**. Has fixed value of **haproxy**
+- **`webengine.system`**. Has fixed value of **haproxy**
+- **`webengine.cluster.name`**. User configured. Enter a name to identify the HAProxy cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`webengine.node.name`**. Has the value of host name of the machine which is being monitored
```
[the remark this answers is not in the document]
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 12 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 11 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 10 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 9 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 8 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 7 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 1 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

File: docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md (hunk 6 above)
Comment (yasar-sumologic) [reply docs/integrations/web-servers/opentelemetry/haproxy-opentelemetry.md]: fixed it.

**Review by JV0812 — APPROVED** [review]
LGTM

**Comment by sumoanema** [comment]
@kimsauce - Could you please have look at this PR.

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
> Could you please have look at this PR.

@sumoanema all set! Looks good.

**Pull request merged**